src/lite-yt-embed.js

```diff
@@ -70,7 +70,7 @@ class LiteYTEmbed extends HTMLElement {
         if (as) {
             linkElem.as = as;
         }
-        linkElem.crossorigin = true;
+        linkElem.crossOrigin = 'anonymous';
```

Comment:
Nice catch on this being set to a boolean and not one of the defined keywords (or set of them.)

I'm slightly confused however since it is [defined in the HTML Spec](https://html.spec.whatwg.org/multipage/semantics.html#attr-link-crossorigin) as `crossorigin`. This is further supported by [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link). Could you please provide some support to your claim that it is required to be a camalcase property? If that is the case then the spec and MDN need to be amended to correct this.

_I am going to request changes here simply to try and make sure this discussion is seen and resolved before merging. Whether or not changes are actually needed is yet to be determined._

Reply:
Just sorted this out in my own head after thinking about your words a bit more. The [HTMLLinkElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLLinkElement) certainly has it defined as `crossOrigin` for JavaScript access.

With this detail confirmed, this patch looks great to me. Nice find.

_Side note, rather than "Setting the attribute name" saying "Setting the property" in some way would have cleared up the bit of confusion here. JS/DOM generally works on object properties. While the attributes are to an HTML element in text form._

Reply:
I found it when using this approach with JSX. The TS types in JSX use camelcase notation for crossOrigin, and a quick look from chrome console I made sure to it.

TS: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0d04fc967106fe48895d1dfdcbde070072a1b153/types/react/index.d.ts#L1837